nncf/quantization/quantize_model.py

```diff
@@ -218,3 +218,26 @@ def quantize_with_accuracy_control(
         )
 
     raise RuntimeError(f"Unsupported type of backend: {backend}")
+
+
+@api(canonical_alias="nncf.weights_compression")
+def weights_compression(model: TModel, use_fake_quantize: bool = False) -> TModel:
+    """
+    Applies per-channel weights compression with dequantization or fake quantize insertion.
+
+    :param model: A model to be compressed.
+    :type  model: TModel
```

Comment:
Our code style guide doesn't require specifying parameter type.
Please remove to make it shorter and more consistent with the current code style.

Reply:
Done

Reply:
This is incorrect comment. API function requires type for params. Please, take a look at https://github.com/openvinotoolkit/nncf/blob/aff8237371baaca0df3ac7c71172425a844fb7c3/nncf/quantization/quantize_model.py#L29

Reply:
ok, havent't noticed. Style guide should be updated then.